config.py

```diff
@@ -9,8 +9,8 @@ def __init__(self, app_type, *cfg_search_path):
         self.app_type = app_type
         super().__init__(*cfg_search_path)
 
-    def load_config_file(self, cfg_file):
-        super().load_config_file(cfg_file)
+    def load_config_file(self, *search_path, env_var=None):
+        super().load_config_file(*search_path, env_var=env_var)
```

Comment:
We're using `env_var` here which works but doesn't really make sense since this Config doesn't inherit from EnvConfiguration. This should probably be changed in EGCG-Core.